aws/resource_aws_rds_cluster_instance.go

```diff
@@ -196,6 +196,12 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				ValidateFunc: validateArn,
 			},
 
+			"multiaz": {
```

Comment:
A few things with this new attribute:
* Can you please document this new argument in `website/docs/r/rds_cluster_instance.html.markdown`?
* Can you please adjust this to be named `multi_az` to match the `aws_db_instance` resource? https://www.terraform.io/docs/providers/aws/r/db_instance.html#multi_az
* Can you please add a new acceptance test in `aws/resource_aws_rds_cluster_instance_test.go` with this new argument set to `false`? Copy paste of an existing test and test configuration is a great way to get started 👍